<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>968</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="0">
          <widget class="QPushButton" name="btn1">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>90</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
background-color: rgb(88, 214, 141 );
border-radius:3px;
color:white
}


QPushButton:hover{
border-radius:0px;
background-color: rgb(130, 224, 170);
border-radius:3px;
color:white
}

QPushButton:pressed{
	border-radius:0px;
background-color: rgb(88, 214, 141 );
border-radius:3px;
color:white
}
</string>
           </property>
           <property name="text">
            <string>BUTTON</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="lab2">
           <property name="font">
            <font>
             <pointsize>30</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: green;</string>
           </property>
           <property name="text">
            <string>ON</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="btn2">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>90</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
background-color: rgb(88, 214, 141 );
border-radius:3px;
color:white
}


QPushButton:hover{
border-radius:0px;
background-color: rgb(130, 224, 170);
border-radius:3px;
color:white
}

QPushButton:pressed{
	border-radius:0px;
background-color: rgb(88, 214, 141 );
border-radius:3px;
color:white
}
</string>
           </property>
           <property name="text">
            <string>BUTTON</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QPushButton" name="btn3">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>90</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
background-color: rgb(88, 214, 141 );
border-radius:3px;
color:white
}


QPushButton:hover{
border-radius:0px;
background-color: rgb(130, 224, 170);
border-radius:3px;
color:white
}

QPushButton:pressed{
	border-radius:0px;
background-color: rgb(88, 214, 141 );
border-radius:3px;
color:white
}
</string>
           </property>
           <property name="text">
            <string>BUTTON</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="lab1">
           <property name="font">
            <font>
             <pointsize>30</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: green;</string>
           </property>
           <property name="text">
            <string>ON</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="lab3">
           <property name="font">
            <font>
             <pointsize>30</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: green;</string>
           </property>
           <property name="text">
            <string>ON</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QPushButton" name="btn5">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>90</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>20</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
background-color: rgb(88, 214, 141 );
border-radius:3px;
color:white
}


QPushButton:hover{
border-radius:0px;
background-color: rgb(130, 224, 170);
border-radius:3px;
color:white
}

QPushButton:pressed{
	border-radius:0px;
background-color: rgb(88, 214, 141 );
border-radius:3px;
color:white
}
</string>
           </property>
           <property name="text">
            <string>BUTTON</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="lab5">
           <property name="font">
            <font>
             <pointsize>30</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: green;</string>
           </property>
           <property name="text">
            <string>ON</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Integrantes:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Anderson Peralta</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Bryam Illisaca</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2" colspan="2">
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Sistema Generador de Fallos</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
